contracts/stake/src/contract.rs

```diff
@@ -460,6 +522,30 @@ impl StakingTrait for Staking {
         }
     }
 
+    fn withdraw_rewards_deprecated(env: Env, sender: Address) {
+        env.storage()
```

Comment:
The old function were supposed to become a deprecated ones.

Reply:
what I understand is `withdraw_rewards` from the current main branch is supposed to become `withdraw_rewards_deprecated` and the function `withdraw_rewards` is from the stake contract tag v1.0.0. Is it vice versa?